<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MyBrowser</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/tbicons/icon.png</normaloff>:/tbicons/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit{
border-radius: 8px;
border: 1px solid black;
selection-color: rgb(255, 255, 127);
}
QLineEdit::hover{
border: 2px solid rgb(255, 255, 127);}
QLineEdit::selected{
border: 2px solid rgb(255, 255, 127);}
</string>
  </property>
  <property name="iconSize">
   <size>
    <width>25</width>
    <height>25</height>
   </size>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="back_button">
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Zurück (Backspace)</string>
        </property>
        <property name="whatsThis">
         <string>Klicken um eine Seite zurück zu gelangen</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/tbicons/img/back.png</normaloff>:/tbicons/img/back.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Backspace</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="forward_button">
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Vor</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/tbicons/img/forward.png</normaloff>:/tbicons/img/forward.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Shift+Backspace</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reload_button">
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Neu Laden/Laden Abbrechen (F5)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/tbicons/img/world.png</normaloff>:/tbicons/img/world.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
        <property name="shortcut">
         <string>F5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="home_button">
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Zur Home Seite (F6)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/tbicons/img/home.png</normaloff>:/tbicons/img/home.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
        <property name="shortcut">
         <string>F6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="toolTip">
         <string>Url Eingabe Zeile</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="but_settings">
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/tbicons/img/setting.png</normaloff>:/tbicons/img/setting.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Alt+S</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="but_book">
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/tbicons/img/show_hide_sidebar.png</normaloff>:/tbicons/img/show_hide_sidebar.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="webvie_layout">
      <item>
       <widget class="QWebView" name="webView">
        <property name="maximumSize">
         <size>
          <width>200000</width>
          <height>200000</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="url">
         <url>
          <string>file:///C:/Qt/2010.05/qt</string>
         </url>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>450</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
        <widget class="QWidget" name="bookmarks">
         <attribute name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/tbicons/img/star.png</normaloff>:/tbicons/img/star.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Lesezeichen</string>
         </attribute>
         <attribute name="toolTip">
          <string>Lesezeichen</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="but_hinzu">
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="res.qrc">
                <normaloff>:/tbicons/img/add.png</normaloff>:/tbicons/img/add.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
              <property name="shortcut">
               <string>Ctrl+Shift+N</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="but_edit">
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="res.qrc">
                <normaloff>:/tbicons/img/edit.png</normaloff>:/tbicons/img/edit.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="but_addFolder">
              <property name="maximumSize">
               <size>
                <width>35</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="res.qrc">
                <normaloff>:/tbicons/img/addFolder.png</normaloff>:/tbicons/img/addFolder.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTreeWidget" name="tree_bookmarks">
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="animated">
             <bool>true</bool>
            </property>
            <property name="columnCount">
             <number>2</number>
            </property>
            <attribute name="headerVisible">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Adresse</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="history">
         <attribute name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/tbicons/img/uhr.png</normaloff>:/tbicons/img/uhr.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Verlauf</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="but_clear">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Alles Löschen</string>
              </property>
              <property name="icon">
               <iconset resource="res.qrc">
                <normaloff>:/tbicons/img/tashCan.png</normaloff>:/tbicons/img/tashCan.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTreeWidget" name="tree_history">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="animated">
             <bool>true</bool>
            </property>
            <property name="headerHidden">
             <bool>true</bool>
            </property>
            <property name="columnCount">
             <number>2</number>
            </property>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string notr="true">2</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QProgressBar" name="barLoadProgress">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>15</height>
         </size>
        </property>
        <property name="value">
         <number>24</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="textVisible">
         <bool>true</bool>
        </property>
        <property name="textDirection">
         <enum>QProgressBar::TopToBottom</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="stat">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>15</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="butDownloadManager">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/tbicons/img/download.png</normaloff>:/tbicons/img/download.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="butSetZoomToNorm">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>100%</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderrSiteZoom">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximum">
         <number>800</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickInterval">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>back_button</tabstop>
  <tabstop>forward_button</tabstop>
  <tabstop>reload_button</tabstop>
  <tabstop>home_button</tabstop>
  <tabstop>but_settings</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>back_button</sender>
   <signal>clicked()</signal>
   <receiver>webView</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>forward_button</sender>
   <signal>clicked()</signal>
   <receiver>webView</receiver>
   <slot>forward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
